nsinit/exec.go

```diff
@@ -51,7 +45,7 @@ func execAction(context *cli.Context) {
 	}
 	process := &libcontainer.Process{
 		Args:   context.Args(),
-		Env:    context.StringSlice("env"),
+		Env:    append(os.Environ(), context.StringSlice("env")...),
```

Comment:
i think this is somewhat unexpected as users now have to clear their env if they want to finely control the exec process's environment variable.